extensions/web3-profile/package.json

```
@@ -0,0 +1,56 @@
+{
+  "$schema": "https://www.raycast.com/schemas/extension.json",
+  "name": "web3-profile",
+  "title": "Web3 Profile",
+  "description": "Search for ENS names and view its profile — Avatar, description, social URLs and more!",
+  "icon": "web3-profile-icon.png",
+  "author": "peduarte",
+  "categories": [
+    "Web"
+  ],
+  "keywords": [
+    "web3",
+    "ens",
+    "ethereum"
+  ],
+  "license": "MIT",
+  "commands": [
+    {
+      "name": "search-web3-profile",
+      "title": "Search Web3 Profiles",
```

```suggestion
      "title": "Search Profiles",
```
Maybe move Web3 to subtitle?